tests/testthat.R

```diff
@@ -6,5 +6,29 @@ cargo_available <- system2("cargo") == 0L
 not_cran <- identical(Sys.getenv("NOT_CRAN"), "true")
 
 if (cargo_available && not_cran) {
+  # This retrieves exrepssions that represent
+  # default arguments of rust_source,
+  # which are used for reference
+  test_env <- formals(rextendr::rust_source)
```

Comment:
Not sure I like the name `test_env`. It sounds more general than it is. How about `rust_source_defaults`?